src/cogs/trivia.py

```diff
@@ -52,6 +57,7 @@ async def on_guess(self, msg: discord.Message):
 
     @commands.slash_command(name='stop_trivia', description='stops the in-progress trivia game in this channel')
     async def stop_trivia(self, ctx: discord.ApplicationContext):
+        await ctx.respond(f"Trivia has been stopped by {ctx.user.mention} {Emotes.DRINKING}")
```

Comment:
does this make sense in the context where Nix stops the trivia game (i.e. from an internal error)